modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/test/UploadFileEntryMVCActionCommandTest.java

```diff
@@ -105,6 +119,26 @@ public void setUp() throws Exception {
 							new HashMap<>()),
 						null, RandomTestUtil.randomString());
 				}));
+
+		User user = DDMFormUtil.getDDMFormDefaultUser(
+			TestPropsValues.getCompanyId());
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId());
+
+		UserLocalServiceUtil.updateStatus(
+			user, WorkflowConstants.STATUS_INACTIVE, serviceContext);
```

Comment:
@JhossephG you don't need to do this. The `getDDMFormDefaultUser` is already updating the status to inactive.